tests/conftest.py

```diff
@@ -393,6 +393,14 @@ def moving_image(remote_sample) -> pathlib.Path:
     return remote_sample("moving_image")
 
 
+@pytest.fixture(scope="session")
+def dfbr_features(remote_sample) -> pathlib.Path:
+    """Sample pytest fixture for DFBR features.
+    Download features for pytest.
+    """
+    return remote_sample("dfbr_features")
```

Comment:
Prefer a descriptive name to acronyms, i.e. what is 'dfbr'. At least explain in the docstring.

Reply:
https://github.com/TissueImageAnalytics/tiatoolbox/blob/4a4290bd4a06835ac1f8e333f4396e764e0555a8/tests/conftest.py#L399-L404